### ISSUE TITLE
State of the maintenance

### ISSUE THREAD

**Issue by azmeuk** [jazzband/django-push-notifications#685]
I am opening this issue to discuss the current state of the maintenance.

I am part of a project that requires dependencies to be active libraries, and I cannot say if django-push-notifications is active, stable or dying. Ideally I would love some of the maintainers or people from @jazzband to share their thoughts here :pray:. This can also be a good place to call for help if new maintainers are needed.

The good things:
- The project is stable, there is no need for active development

The bad omens:
- The last release was [18 months](https://github.com/jazzband/django-push-notifications/releases/tag/3.0.0) ago, the one before was [3 years ago](https://github.com/jazzband/django-push-notifications/releases/tag/2.0.0), and the one before was [7 years ago](https://github.com/jazzband/django-push-notifications/releases/tag/1.4.1)
- 134 issues are opened, in the past 12 months only 2 messages were written by people from jazzband
- Several issues about technical deprecation are not answered/fixed/closed: #546 #618 #622 #675
- Several PR fixing those issues are not reviewed : #615 #674
- Uncontroversial fixes are not merged: #669 #672 #678

To summarize:

- Do you maintainers still have time to spend on this project? 
- Do you need help with the maintenance?
- Would you recommend using this library?
- Would you recommend using alternatives?

cc @jamaalscarlett @Proper-Job @antwan 
Related to #205

**Comment by koedal** [comment]
Thanks for posting this. It sure would be nice to know if the maintainers are evening considering fixing any of those issue.

**Comment by jamaalscarlett** [comment]
@azmeuk Thank you for this.  Part of the reason @jleclanche moved the project to jazzband was to have more people able to review and merge PRs.  I am currently reviewing PRs and issues, but I am a full-time dev and I don't have the bandwidth to manage the project by myself.

**Comment by azmeuk** [comment]
@jamaalscarlett thank you for your answer. To be clear, can this project be considered unmaintained at the moment?

With a quick look to the current issues, I see that the most problematic issue is #622, that blocks python 3.10 and python 3.11 users to use the library. A lot of the webpush issues are addressed by #674, so this is a good thing.
With python 3.10+ and WP Safari support, I suppose the maintenance need would be pretty low.

**Comment by jleclanche** [comment]
@azmeuk AFAIK this is not maintained very actively but again, that's why it's in [Jazzband](https://jazzband.co/), to make it easier for new maintainers to onboard without extra assistance.

**Comment by jamaalscarlett** [comment]
> @azmeuk AFAIK this is not maintained very actively but again, that's why it's in [Jazzband](https://jazzband.co/), to make it easier for new maintainers to onboard without extra assistance.

Agreed.  I am going to take a swing at updating FCM support to the new V1 api.  I have never used the apns notifications so that work needs some volunteers.

**Comment by jamaalscarlett** [comment]
@azmeuk Version 3.0.2 has been released.  Thank you @blighj for the webpush enhancements!
